package.json

```diff
@@ -1,68 +1,65 @@
 {
-  "name": "wherobots-sql-driver",
-  "version": "0.4.2",
-  "description": "TypeScript SDK for Wherobots DB",
-  "license": "Apache-2.0",
-  "main": "dist/src/index.js",
-  "keywords": [
-    "wherobots",
-    "sedona",
-    "geospatial",
-    "spatial sql"
-  ],
-  "homepage": "https://github.com/wherobots/wherobots-typescript-sdk",
-  "bugs": "https://github.com/wherobots/wherobots-typescript-sdk/issues",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/wherobots/wherobots-typescript-sdk"
-  },
-  "engines": {
-    "node": ">=18.0.0"
-  },
-  "scripts": {
-    "build": "tsc",
-    "build:check": "tsc --noEmit",
-    "lint": "eslint .",
-    "lint:fix": "eslint . --fix",
-    "format": "prettier --check .",
-    "format:fix": "prettier --write .",
-    "prepare": "husky",
-    "prepublish": "npm run build",
-    "test": "vitest"
-  },
-  "devDependencies": {
-    "@commitlint/cli": "^19.4.0",
-    "@commitlint/config-conventional": "^19.2.2",
-    "@swc-node/register": "^1.10.9",
-    "@tsconfig/node-lts": "^20.1.3",
-    "@tsconfig/strictest": "^2.0.5",
-    "@types/uuid": "^10.0.0",
-    "@types/ws": "^8.5.12",
-    "@typescript-eslint/eslint-plugin": "^7.18.0",
-    "@typescript-eslint/parser": "^7.18.0",
-    "eslint": "^8.57.0",
-    "eslint-config-airbnb": "^19.0.4",
-    "eslint-config-airbnb-typescript": "^18.0.0",
-    "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^5.2.1",
-    "eslint-plugin-security": "^3.0.1",
-    "husky": "^9.1.4",
-    "lint-staged": "^15.2.9",
-    "prettier": "^3.3.3",
-    "typescript": "^5.5.4",
-    "vitest": "^3.0.5",
-    "vitest-fetch-mock": "^0.3.0"
-  },
-  "dependencies": {
-    "apache-arrow": "^17.0.0",
-    "bufferutil": "^4.0.8",
-    "cbor": "^9.0.2",
-    "pino": "^9.3.2",
-    "pino-pretty": "^11.2.2",
-    "read-pkg-up": "^7.0.0",
-    "uuid": "^10.0.0",
-    "ws": "^8.18.0",
-    "zod": "^3.23.8"
-  }
+    "name": "wherobots-sql-driver",
+    "version": "0.4.2",
+    "description": "TypeScript SDK for Wherobots DB",
+    "license": "Apache-2.0",
+    "main": "dist/src/index.js",
+    "type": "module",
```

Comment:
is `"type": "module"` necessary? it breaks the running of the example scripts (e.g. `node -r @swc-node/register examples/connectionWithDefaults.ts`)

Reply:
I needed to use it to be esm compatible for one of the packages. Will see if I can remove it. Also, I recommend we use npx for this so I have now replaced the command with npx tsx examples/connectionWithDefaults.ts (tsx runs ts files)